test_specs/features/step_definitions/API_steps.rb

```diff
@@ -23,21 +22,128 @@
   end
 end
 
+Given(/^I post an idea on "([^"]*)" community with Title: "([^"]*)", Category: "([^"]*)", Tags: "([^"]*)" and "([^"]*)" parameters$/) do |site, idea_title, category_title, tags, params|
+# Get response for the Rest API Call and print it raw
```

Comment:
We have to understand if we want to pass everything raw to idea or, if we want to get the other values in advance or hardcode them. It is not the responsibility of this test call the other APIs